code/game/atoms_movable.dm

```diff
@@ -177,11 +177,11 @@
 
 /atom/movable/proc/stop_pulling()
 	if(pulling)
-		pulling.pulledby = null
-		if(ismob(usr))
-			log_combat(usr, pulling, "has stopped pulling", addition = "at [AREACOORD(usr)]")
+		if(ismob(pulling?.pulledby))
+			pulling.pulledby.log_message("has stopped pulling [key_name(pulling)] at", LOG_ATTACK)
 		if(ismob(pulling))
-			log_combat(pulling, usr, "stopped being pulled by", addition = "at [AREACOORD(pulling)]")
+			pulling.log_message("has stopped being pulled by [key_name(pulling.pulledby)] at", LOG_ATTACK)
```

Comment:
The end of the sentance shouldn't contain at. It's up to the logging displayer to render it how it wants and position will usually be drawn on the left rather than on the right of the log message.

Reply:
Adressed